### ISSUE TITLE
ESP Distance, how to modify it?

### ISSUE THREAD

**Issue by java17lzh** [danielkrupinski/Osiris#3817]
ESP Distance, how to modify it?

**Comment by zanex78** [comment]
I dont think this is posible since esp distance depends on the csgo render distance and I dont know any posible way to modify it without it being detected as a "valid game file"

**Comment by martinjanas** [comment]
Calculate distance from local origin to enemy origin usint distTo();

Insert this where is your esp loop:

if (distance >= max_distance)
    continue;

You can only shorten the distance btw.
